doc-content/shared-kie-docs/src/main/asciidoc/KieServer/controller-java-api-con.adoc

```diff
@@ -0,0 +1,189 @@
+[id='controller-java-api-con_{context}']
+= {CONTROLLER} Java client API for {KIE_SERVER} templates and instances
+
+{PRODUCT} provides a {CONTROLLER} Java client API that enables you to connect to the {CONTROLLER} using REST or WebSocket protocol from your Java client application. You can use the {CONTROLLER} Java client API as an alternative to the {CONTROLLER} REST API to interact with your {KIE_SERVER} templates (configurations), {KIE_SERVER} instances (remote servers), and associated KIE containers (deployment units) in {PRODUCT} without using the {CENTRAL} user interface. This API support enables you to maintain your {PRODUCT} servers and resources more efficiently and optimize your integration and development with {PRODUCT}.
```

Comment:
..using the REST or WebSocket...

Reply:
Nah, I had "the REST ..." before and sounds odd. It's just REST/WebSocket protocol, not the REST/WebSocket protocol.